bundle/file.go

```diff
@@ -24,6 +24,42 @@ type Descriptor struct {
 	closeOnce *sync.Once
 }
 
+// LazyFile defers reading the file until the first call of Read
+type LazyFile struct {
```

Comment:
We shouldn't need to export this, I think.

Reply:
Agreed. The thought process was since it provides general lazy file reading that it may be needed by other components but as that isn't currently the case I can lowercase the L when I get to a computer. Thanks for the review!

Reply:
Thanks for contributing. Would you mind also fixing the lowercase of the corresponding comments?

Reply:
done